ScrumPoker.Data/Data/GameRoomDataBase.cs

```diff
@@ -0,0 +1,47 @@
+using ScrumPoker.Core.Models;
+using ScrumPoker.DataBase.Interfaces;
+
+namespace ScrumPoker.Data.Data;
+
+public class GameRoomDataBase : IGameRoomData
+{
+    private static List<GameRoom> _gameRooms = new List<GameRoom>();
+    private static int Id { get; set; }
+    
+    public void Create(string name)
+    {
+        var gameRoom = new GameRoom
+        {
+            Name = name,
+            Id = ++Id
+        };
+        
+        _gameRooms.Add(gameRoom);
+    }
+
+    public List<GameRoom> GetAllGameRooms()
```

Comment:
Naming of methods is mixed here.
Either include "GameGoom" in method name or don't.
I suggest that you don't because it's implied by class name.
Have methods like `Create`, `GetAll`, etc.